galite-core/src/main/kotlin/org/kopi/galite/ui/vaadin/actor/VActorNavigationItem.kt

```diff
@@ -40,20 +41,21 @@ class VActorNavigationItem(text: String,
                            acceleratorKey: Key?,
                            keyModifier: KeyModifier?,
                            icon: Any?,
+                           val navigationmenu : VNavigationMenu,
```

Comment:
need to check DWindow :  add navigationMenu in addActorsToGUI method
`panel.addActor(actor, actorDef, navigationMenu)`

& change val bname to **navigationMenu**